frontend/workflows/sourcecontrol/src/create-repository.tsx

```diff
@@ -113,18 +102,25 @@ const Confirm: React.FC<WizardChild> = () => {
 
   return (
     <WizardStep error={repoData.error} isLoading={repoData.isLoading}>
-      <Confirmation action="Pull Request" />
-      <MetadataTable data={[{ name: "Url", value: instance.data?.url }]} />
+      <Confirmation action="Repository Creation">
+        <ConfirmationMessage>
+          Your new repository can be found&nbsp; <Link href={instance.data.url}>here</Link>
+        </ConfirmationMessage>
+      </Confirmation>
     </WizardStep>
   );
 };
 
-const CreateRepository: React.FC<WorkflowProps> = ({ heading, options }) => {
+const CreateRepository: React.FC<WorkflowProps> = ({ heading }) => {
   const dataLayout = {
-    extraOptionsData: {},
-    resourceData: { cache: false },
+    repositoryData: {
+      cache: false,
+      hydrator: () => {
+        return client.post("/v1/sourcecontrol/getRepositoryOptions");
```

Comment:
could the github mocks in the backend be updated?  https://github.com/lyft/clutch/blob/main/backend/mock/service/githubmock/githubmock.go, I think it's just missing `GetRepositoryOptions`. Just so that we can easily test the frontend